src/lib/webpack-config.js

```diff
@@ -433,6 +435,7 @@ const production = config => addPlugins([
 		minify: true,
 		stripPrefix: config.cwd,
 		staticFileGlobsIgnorePatterns: [
+			/polyfills\.js$/,
```

Comment:
Did you ignore this because all `serviceWorker`-compatible browsers don't need a `fetch` or `Promise` polyfill?

_(^ I'm not certain that that's 100% true)_